### ISSUE TITLE
tableplus says im still using unregistered version even after buying a license and registering.

### ISSUE THREAD

**Issue by jonesjsc** [TablePlus/TablePlus-Windows#435]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):

3. The steps to reproduce this issue:

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by jonesjsc** [comment]
TablePlus Buil;d 176 says that I'm still using a trial version of the software even after registering.  
I was using tableplus - bumped in to the freeware limit of having 3 things opened at once.  I purchased the license and applied the license.  This limit was lifted but i happened to notice that on the Welcome screen it still say i was uing the trial version.
Of course when I exited out and then relaunched that notification was cleared.  
For sure this is no big deal just though I'd mention it.
